modules/vaos/spec/services/common/jwt_wrapper_spec.rb

```diff
@@ -1,141 +1,111 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require_relative '../../../app/services/common/jwt_wrapper'
+require_relative '../../support/fixture_helper'
```

Comment:
You should be able to just revert to the original code here too and not make any changes.